cdc/redo/manager.go

```diff
@@ -153,24 +148,35 @@ func NewManager(ctx context.Context, cfg *config.ConsistentConfig, opts *Manager
 	if cfg == nil || ConsistentLevelType(cfg.Level) == ConsistentLevelNone {
 		return &ManagerImpl{enabled: false}, nil
 	}
+	if cfg.FlushIntervalInMs > flushIntervalInMs {
+		flushIntervalInMs = cfg.FlushIntervalInMs
+	}
```

Comment:
- `FlushIntervalInMs` is not used in `FileWriterConfig` any more
- Does this mean the minimum flush interval is 2s?

Reply:
Yes for the time being, we will adjust this parameter according to the later test.